client/src/modules/stock/entry/entry.html

```diff
@@ -2,12 +2,32 @@
   <div class="bhima-title">
     <ol class="headercrumb">
       <li class="static" translate>TREE.STOCK</li>
-      <li class="title"><span translate>STOCK.ENTRY</span> <span class="text-uppercase badge badge-primary">{{StockCtrl.movement.entry_type}}</span></li>
+      <li ng-class="{ 'title' : !StockCtrl.depot.uuid, 'static' : StockCtrl.depot.uuid }">
+        <span translate>STOCK.ENTRY</span>
+      </li>
+
+      <li class="title" ng-if="StockCtrl.depot.uuid">
```

Comment:
Why not ```StockCtrl.selectedDepot```?

Reply:
I believe the previous version used `depot`, so I was attempting not to change the bindings too much :(.